go/vt/vtgate/vindexes/vindex.go

```diff
@@ -57,32 +57,33 @@ type Vindex interface {
 	// Which means Map() maps to either a KeyRange or a single KeyspaceID.
 	IsUnique() bool
 
-	// IsFunctional returns true if the Vindex can compute
```

Comment:
What is the reason for removing this from the interface?

Reply:
Reducing the surface area. There was probably a time when this interface was used more often. But now, I saw it being used only in one place, and it wasn't necessary.